collect_app/src/main/res/layout/project_settings_dialog_layout.xml

```diff
@@ -76,25 +75,26 @@
         android:layout_width="match_parent"
         android:layout_height="1dp"
         android:layout_marginTop="@dimen/margin_small"
-        android:background="?colorPrimary"
+        android:background="?dividerColor"
         app:layout_constraintStart_toEndOf="parent"
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toBottomOf="@+id/admin_settings_button" />
 
     <LinearLayout
         android:id="@+id/project_list"
-        android:layout_width="match_parent"
+        android:layout_width="0dp"
         android:layout_height="wrap_content"
         android:orientation="vertical"
         app:layout_constraintStart_toStartOf="@+id/guideline_start"
+        app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintTop_toBottomOf="@+id/top_divider" />
 
     <View
         android:id="@+id/bottom_divider"
         android:layout_width="match_parent"
         android:layout_height="1dp"
         android:layout_marginTop="@dimen/margin_small"
-        android:background="?colorPrimary"
+        android:background="?dividerColor"
```

Comment:
I think we should use a Material theme attribute here rather than dividerColor which is a custom attribute (I'd like to get rid of as well). This should be a colorOnSurface I think.

Reply:
I'm not sure about it, then it would be very thick:
![Screenshot_1618224683](https://user-images.githubusercontent.com/3276264/114384458-0a473700-9b8f-11eb-8989-e4173bec7155.png)
It doesn't look like the divider in Google dialogs.

Reply:
maybe we should use it + transparency or something?

Reply:
ok I did it using transparency https://github.com/getodk/collect/pull/4510/commits/7cf918af33d1bc1d771b128e92266e2397dd1ef3 and it looks fine.